src/platform/site-wide/sass/modules/_m-crisis-line.scss

```diff
@@ -103,7 +98,7 @@ button.va-crisis-line {
     max-height: 100%;
   }
   &-title {
-    color: #212121;
+    color: $color-base;
```

Comment:
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L51